notebooks/data-sources/moc_openstack_demo.ipynb

```diff
@@ -0,0 +1,383 @@
+{
```

Comment:
<p>try this <code>&nbsp;unauth_connection = boto3.client('s3', endpoint_url=s3_endpoint_url, config=Config(signature_version=UNSIGNED))</code></p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aicoe-aiops/ocp-ci-analysis/pull/444/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>